test/infra/infra-mysql84-binlog/docker-compose-init.bash

```
@@ -0,0 +1,136 @@
+COMPOSE_CMD="docker compose"
+if ! $COMPOSE_CMD version &>/dev/null; then
+    COMPOSE_CMD="docker-compose"
+    if ! $COMPOSE_CMD version &>/dev/null; then
+        echo "ERROR: Neither 'docker compose' nor 'docker-compose' found!"
+        exit 1
+    fi
+fi
+
+if [ -z "${INFRA_ID}" ]; then echo "Error: INFRA_ID must be set"; exit 1; fi
+
+export ROOT_PASSWORD=$(echo -n "${INFRA_ID}" | sha256sum | head -c 10)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)